### PR TITLE
feat: Add renovatebot config

### LINKED ISSUE #46: Add renovate for auto-dep updates

### PR DESCRIPTION
**Pull request by joaopedrodcf**
This PR adds a config for renovatebot so that it opens PRs of minors and patches in a bulk and the majors as separate PRs, as if they follow server the major versions are probably the only ones that may require us to change

Closes #46

### CODE REVIEW

**Comment by joaopedrodcf** [comment]
@SoaresMG @rubensilva3

**Comment by SoaresMG** [comment]
It seems that by default it brings config/recommended

And maybe we should also add $schema wdyt?

**Comment by SoaresMG** [comment]
We should also look at renovate commit msgs, for some reason the bot initially created the [commit](https://github.com/SoaresMG/nuxt-mock-server/pull/58) without conventional-commits.

https://docs.renovatebot.com/semantic-commits/

**Comment by joaopedrodcf** [comment]
Added the config to use semantic commits @SoaresMG

File: yarn.lock [hunk 1]
```diff
@@ -486,7 +486,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@changesets/changelog-github@npm:^0.5.0":
```
Comment (joaopedrodcf) [inline yarn.lock]: Pinned all dependencies

**Comment by joaopedrodcf** [comment]
Before merging validate if everything is still working 🙏

**Review by SoaresMG — DISMISSED** [review]

**Review by SoaresMG — APPROVED** [review]

**Pull request merged**